### PR TITLE
Updates to include latest iOS OSS projects

### PR DESCRIPTION
**Pull request by dfed**
Cc @JakeWharton @alanf @ericmuller22

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -202,6 +202,28 @@ <h4 class="menu-item-name">SocketRocket</h4>
...
+              <li class="menu-item">
+                <div class="menu-item-info">
+                  <h4 class="menu-item-name">Aardvark</h4>
+                  <span class="menu-item-price"><a href="https://github.com/square/Aardvark" target="_blank">GitHub</a></span>
+                  <p class="menu-item-description">Make every bug actionable.</p>
+                </div>
+              </li>
+              <li class="menu-item">
+                <div class="menu-item-info">
+                  <h4 class="menu-item-name">[Space Commander]</h4>
```
Comment (alanf) [inline index.html]: `[ Space Commander]` (I know, sorry)

**Comment by alanf** [comment]
Can I provide a Square, rotated image for this? I'd be happy to whip one up

**Comment by dfed** [comment]
@alanf please do!

**Comment by alanf** [comment]
squareimage

**Comment by alanf** [comment]
squareimage

**Comment by alanf** [comment]
LGTM

**Comment by dfed** [comment]
cc @eburke

**Comment by puls** [comment]
I think `index.html` is generated by the `generate.py` script.

**Comment by dfed** [comment]
Thanks @puls... doing it the proper way now. *facepalm* for not reading README.md

**Comment by JakeWharton** [comment]
Sorry for delay. Was DOIN' THANGs.

**Pull request merged**